environment.yml

```diff
@@ -1,138 +1,203 @@
   - pyaudio=0.2.11
+  - pycparser=2.20
+  - pygments=2.6.1
+  - pyjwt=1.7.1
+  - pyopenssl=19.1.0
+  - pyparsing=2.4.7
+  - pyrsistent=0.16.0
+  - pysocks=1.7.1
   - python=3.7.7
+  - python-dateutil=2.8.1
+  - python_abi=3.7
+  - pytorch=1.5.0
+  - pyzmq=19.0.1
   - readline=8.0
-  - six=1.14.0
+  - requests-oauthlib=1.3.0
+  - rsa=4.0
+  - send2trash=1.5.0
+  - six=1.15.0
   - sqlite=3.31.1
+  - tbb=2020.0
+  - tensorboard-plugin-wit=1.6.0
+  - tensorflow-base=2.2.0
+  - terminado=0.8.3
+  - testpath=0.4.4
   - tk=8.6.8
+  - torchaudio=0.5.0
   - torchvision=0.6.0
-  - tqdm=4.45.0
+  - tornado=6.0.4
+  - traitlets=4.3.3
+  - wcwidth=0.2.5
+  - webencodings=0.5.1
+  - werkzeug=1.0.1
+  - wrapt=1.12.1
   - xz=5.2.5
+  - zeromq=4.3.2
+  - zipp=3.1.0
   - zlib=1.2.11
   - zstd=1.3.7
   - pip:
-    - absl-py==0.9.0
-    - alembic==1.4.2
-    - astor==0.8.1
-    - attrdict==2.0.1
-    - audioread==2.1.8
-    - beautifulsoup4==4.9.0
-    - bs4==0.0.1
-    - cachetools==4.1.0
-    - certifi==2020.4.5.1
-    - cffi==1.14.0
-    - chardet==3.0.4
-    - cliff==3.1.0
-    - cmaes==0.5.0
-    - cmd2==0.8.9
-    - coloredlogs==14.0
-    - colorlog==4.1.0
-    - cycler==0.10.0
-    - decorator==4.4.2
-    - deepspeech-gpu==0.7.0
-    - deepspeech-training==0.7.1a0
-    - ds-ctcdecoder==0.7.1a0
-    - future==0.18.2
-    - gast==0.2.2
-    - google-auth==1.14.1
-    - google-auth-oauthlib==0.4.1
-    - google-pasta==0.2.0
-    - grpcio==1.28.1
-    - h5py==2.10.0
-    - humanfriendly==8.2
-    - idna==2.9
-    - joblib==0.14.1
-    - keras-applications==1.0.8
-    - keras-preprocessing==1.1.0
-    - kiwisolver==1.2.0
-    - librosa==0.7.2
-    - llvmlite==0.31.0
-    - mako==1.1.2
-    - markdown==3.2.1
-    - markupsafe==1.1.1
-    - matplotlib==3.2.1
-    - numba==0.47.0
-    - numpy==1.18.3
-    - oauthlib==3.1.0
-    - opt-einsum==3.2.1
-    - optuna==1.3.0
-    - opuslib==2.0.0
-    - pandas==1.0.3
-    - pbr==5.4.5
-    - praat-textgrids==1.3.1
-    - prettytable==0.7.2
-    - progressbar2==3.51.3
-    - protobuf==3.11.3
-    - pyasn1==0.4.8
-    - pyasn1-modules==0.2.8
-    - pycparser==2.20
-    - pydantic==1.5.1
-    - pyparsing==2.4.7
-    - pyperclip==1.8.0
-    - python-dateutil==2.8.1
-    - python-editor==1.0.4
-    - python-utils==2.4.0
-    - pytz==2020.1
-    - pyxdg==0.26
-    - pyyaml==5.3.1
-    - requests==2.23.0
-    - requests-oauthlib==1.3.0
-    - resampy==0.2.2
-    - rsa==4.0
-    - scikit-learn==0.22.2.post1
-    - scipy==1.4.1
-    - semver==2.9.1
-    - setuptools==46.1.3
-    - soundfile==0.10.3.post1
-    - soupsieve==2.0
-    - sox==1.3.7
-    - sqlalchemy==1.3.16
-    - stevedore==1.32.0
-    - tensorboard==1.15.0
-    - tensorboard-plugin-wit==1.6.0.post3
-    - tensorflow==1.15.2
-    - tensorflow-estimator==1.15.1
-    - termcolor==1.1.0
-    - torchaudio==0.5.0
-    - urllib3==1.25.9
-    - wcwidth==0.1.9
-    - webrtcvad==2.0.10
-    - werkzeug==1.0.1
-    - wheel==0.34.2
-    - wrapt==1.12.1
+    - alembic=
```

Comment:
Why drop the versions?

Reply:
it was the script that did it wrong
I will fix ti